CI failure due to PEP 8 issue:

```
2017-02-20 17:45:13 ERROR: PEP 8: lib/ansible/modules/network/dellos6/dellos6_command.py:179:1: W293 blank line contains whitespace (current)
```

--- a/lib/ansible/modules/network/dellos6/dellos6_command.py
+++ b/lib/ansible/modules/network/dellos6/dellos6_command.py
@@ -176,7 +176,6 @@ def main():
     module = NetworkModule(argument_spec=spec,
                            connect_on_load=False,
                            supports_check_mode=True)
-  
     commands = list(parse_commands(module))
     conditionals = module.params['wait_for'] or list()
 

